libglusterfs/src/glusterfs/common-utils.h

```diff
@@ -1181,6 +1181,13 @@ gf_fop_int(char *fop);
 char *
 get_ip_from_addrinfo(struct addrinfo *addr, char **ip);
 
+int
+close_fds_except_custom(int *fdv, size_t count, void *prm,
+                        void closer(int fd, void *prm));
+
+void
+closer_close(int fd, void *prm);
+
```

Comment:
It should not be exported, it belongs to the of the implementation of close_fds_except().